myTasks/spec/system/tasks_spec.rb

```diff
@@ -27,9 +27,9 @@
           fill_in 'task[description]', with: 'this is new task'
           fill_in 'task[end_date]', with: '2021-06-24'
           fill_in 'task[priority]', with: 1
-          click_button 'create'
+          click_button I18n.t('helpers.submit.create')
           expect(current_path).to eq tasks_path
-          expect(page).to have_content 'Could not create the task.'
+          expect(page).to have_content I18n.t('flash.create_failure')
```

Comment:
ちゃんとテストしてていい感じ！
ただ、Rspecのようなテスト内ではI18nのようなクラスで文字列の存在は確認しないようにお願いしますー
例えばですが、`I18n.t('flash.create_failureeee')` のようにviewファイル内でもRspec内でも両方間違えて書いていた場合(viewファイルからコピペしてテストを書くことは往々にある)、意図していない文字列が表示されている恐れがあるため、画面に表示されている文字が本当に表示されているか確認するようにお願いしますmm
click_button も同じようにお願いします
```suggestion
          expect(page).to have_content '作成に失敗しました'
```

Reply:
なるほど、確かにテストでは実際に表示されている文字のほうがいいですね。
修正します！
そうなると、テスト時は日本語前提ってことですかね？

Reply:
設定を切り替えて両方テストすることもできると思いますが、とりあえず日本語だけで進めましょっか！